docs/readmes/orc8r/deploy_using_juju.md

```diff
@@ -49,12 +49,11 @@ applications:
       domain: <your domain name>
   tls-certificates-operator:
     options:
-      certificate: <your base64 encoded certificate>
-      private-key: <your base64 encoded private key>
-      ca-certificate: <your base64 encoded ca certificate>
+      generate-self-signed-certificates: true
+      ca-common-name: rootca.<your domain name>
 ```
 
-Replace the values in angle brackets.
+> **Warning**: This configuration is unsecure because it uses self-signed certificates.
```

Comment:
I did not know, that the syntax
```markdown
> **Warning**
```
will cause a special rendering in Github. This is magic. :magic_wand: 

FYI Also see https://github.com/community/community/discussions/16925. :wink: